handler/openid/strategy_jwt.go

```diff
@@ -143,10 +144,9 @@ func (h DefaultStrategy) GenerateIDToken(_ context.Context, _ *http.Request, req
 
 	nonce := requester.GetRequestForm().Get("nonce")
 	// OPTIONAL. String value used to associate a Client session with an ID Token, and to mitigate replay attacks.
-	// Although optional, this is considered good practice and therefore enforced.
 	if len(nonce) < fosite.MinParameterEntropy {
 		// We're assuming that using less then 8 characters for the state can not be considered "unguessable"
-		return "", errors.WithStack(fosite.ErrInsufficientEntropy)
+		fmt.Println("WARNING: nonce token is missing or contains insufficient entropy")
```

Comment:
Instead of this, the check should be:

```
if len(nonce) == 0 {
  // skip
} else if len(nonce) < fosite.MinParameterEntropy {
  return "", errors.WithStack(fosite.ErrInsufficientEntropy)
}
```